src/fastoad/module_management/_bundle_loader.py

```diff
@@ -371,8 +383,19 @@ def _install_python_package(self, package_name: str) -> Tuple[Set[Bundle], Set[s
                         try:
                             bundle = self.context.install_bundle(item_package[:-3])
                             bundles.add(bundle)
-                        except BundleException:
-                            failed.add(package_name)
+                        except BundleException as e:
+                            failed.add(item_package[:-3])
+                            if not header_printed:
+                                _LOGGER.warning("=" * 80)
+                                _LOGGER.warning("    %s", package_name)
+                                _LOGGER.warning("-" * 80)
+                                header_printed = True
+                            _LOGGER.warning(
+                                "\n!! Exception: %s\n\n  >> Full error traceback:\n%s",
+                                e,
+                                traceback.format_exc(),
```

Comment:
Is there more than one format ? If yes, have you looked into one that would give a "cleaner" look ?